src/sentry/issues/issue_occurrence.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import hashlib
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any, Mapping, Optional, Sequence
+
+from dateutil.parser import parse as parse_date
+
+from sentry import nodestore
+from sentry.types.issues import GroupType
+from sentry.utils.dates import ensure_aware
+
+
+@dataclass(frozen=True)
+class IssueEvidence:
+    name: str
+    value: str
+    important: bool
```

Comment:
Hmmm, maybe this was discussed in the PR comments but would be nice to comment what 'important' means here.